jmh-benchmarks/README.md

```diff
@@ -0,0 +1,63 @@
+###JMH-Benchmark module
```

Comment:
One more thing: do we want to call this `jmh-benchmarks` or simply `benchmarks`? I can see pros/cons to either option, but interested in your thoughts.